git-config/src/values.rs

```diff
@@ -785,18 +714,12 @@ impl Serialize for Color {
     }
 }
 
-/// Discriminating enum for [`Color`] parsing.
```

Comment:
I thought it was a good thing to discriminate source of errors a little, and would believe this bare enum could be better with `quick-error`. Such an upgrade doesn't have to be done here, but replacing it with `()` seems like a step in the wrong direction.